CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.8.5-RELEASE  (2021-11-04)
```

Comment:
@tikisailor, will you add a new release line here? Same format, and include a summary of the hotfix?